sig-and-wg/wg-knative/knative-eventing-default-pubsub.md

```diff
@@ -0,0 +1,86 @@
+# Overview
+
+As an operator, I want to configure a default messaging middleware that my Kyma cluster should use for eventing.
+
+As a developer, I just want to configure a subscription for an event type from a source in Kyma and do not care about the underlying details.
+
+## Requirement
+
+* The pluggable Knative layer for the messaging middleware (Knative ClusterChannelProvisioner) is available and deployable.
+* Operator will provision the pluggable layer into Kyma (Knative ClusterChannelProvisioner).
+  > Note: There will be some refernce examples that customers can follow. The provisioning/deployment will be an operator action as customes can choose a implementation specific to their needs.
+* Kyma is updated to use Knative version 0.4+.
+
+## The out-of-the-box Kyma installation
+
+The OOB Kyma installation will have NATS Streaming set as the default messaging middleware. The required Knative configurations will be set to use that.
+
+## When to specify the default?
+
+### Post Installation
+
+* Operator either creates a PubSub instance or use existing one.
+
+  >Note: For instance creation, she can use service catalog. Ths is not a must.
+
+* Operator deploys the cluster channel provisioner and injects the secrets.
+
+* Use [Knative semantics](https://github.com/knative/docs/blob/master/eventing/channels/default-channels.md#setting-the-default-channel-configuration) to specify the default ClusterChannelProvisioner.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: default-channel-webhook
+  namespace: knative-eventing
+data:
+  default-channel-config: |
+    clusterdefault:
+      apiversion: eventing.knative.dev/v1alpha1
+      kind: ClusterChannelProvisioner
+      name: cloud-pubsub
+    namespacedefaults:
+      some-namespace:
+        apiversion: eventing.knative.dev/v1alpha1
+        kind: ClusterChannelProvisioner
+        name: cloud-pubsub
+```
+
+  * There will a separate concept where the operator will be able to specify a non-default messaging middleware.
+
+### Why not during installation?
+
+* There is no standard way to deploy a provisioner. It can be done via Helm Chart, plain kubernetes deployments or some other mechanism. 
```

Comment:
```suggestion
* There is no standard way to deploy a provisioner. It can be done via Helm Chart, plain Kubernetes deployments or other mechanisms. 
```